rewrite-java-test/src/test/java/org/openrewrite/java/cleanup/ReplaceLambdaWithMethodReferenceTest.java

```diff
@@ -697,8 +697,8 @@ public interface MyFunction {
               package abc;
                             
               class M {
-                  MyFunction getFunction(String fcn) {
-                      return () -> fcn;
+                  MyFunction getFunction(String function_) {
+                      return () -> function_;
```

Comment:
probably? (the `n` could be for `name` or something else... it's hard to tell w/ such little context)